packages/arb-bridge-eth/contracts/bridge/NitroMigrator.sol

```diff
@@ -162,6 +119,9 @@ contract NitroMigrator is OwnableUpgradeable, IMessageProvider {
         // this returns a different magic value so we can differentiate the user and admin facets
         require(_rollup.isNitroReady() == 0xa4b2, "ADMIN_ROLLUP_NOT_NITRO_READY");
 
+        UpgradeableBeacon beacon = _rollup.nodeFactory().beacon();
+        require(Node(address(beacon)).isNitroReady() == 0xa4b3, "NODE_BEACON_OLD");
```

Comment:
This (and checks below) should be using the `NitroReadyMagicNums` constants

Reply:
oops, pushed